sim2root/CoREASRawRoot/CoreasToRawROOT.py

```diff
@@ -177,10 +177,10 @@ def CoreasToRawRoot(file, simID=None):
   NucleonEnergyCut  = ecuts[0]
   MesonEnergyCut    = HadronEnergyCut # mesons are hadronic, so this should be fine
 
-  parallel = read_list_of_params(inp_input, "PARALLEL") # COREAS-only
+  #parallel = read_list_of_params(inp_input, "PARALLEL") # COREAS-only
```

Comment:
why is this line commented?. Note that there are several changes introduced in this file on the dev_sim2root branch that fix bugs that where present on this file, in particular core position, antenna positions, azimuth and more.
We might need to consider incorporating them for testing. Maybe discuss with @lguelzow and @lwpiotr

Reply:
Ok fixed

Reply:
I mark this as solved, and we will look at the results once we merge sim2root branch.